src/Features/VisualBasic/Portable/EmbeddedLanguages/VisualBasicRegexEmbeddedLanguageClassifier.vb

```diff
@@ -4,12 +4,12 @@
 
 Imports System.Composition
 Imports Microsoft.CodeAnalysis.Classification
-Imports Microsoft.CodeAnalysis.Features.EmbeddedLanguages.Json.LanguageServices
 Imports Microsoft.CodeAnalysis.Features.EmbeddedLanguages.RegularExpressions.LanguageServices
 Imports Microsoft.CodeAnalysis.Host.Mef
 Imports Microsoft.CodeAnalysis.VisualBasic.EmbeddedLanguages.LanguageServices
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.Features.EmbeddedLanguages
+    <ExtensionOrder(Before:=PredefinedEmbeddedLanguageClassifierNames.Json)>
```

Comment:
Do we want the same comment here that we put on the C# implementation?

Reply:
sure.  i can add that in the followup.